commit 9acaf3feb0ebd0987945bd6663f379491ecd1613 (refs/changes/33/2233/1)
Author: Richard Bradley <richard.bradley@joyent.com>
Date:   2017-07-18T10:35:16+00:00 (2 years, 3 months ago)
    
    MANATEE-304 manatee doesn't always come up with very large datasets

diff --git a/lib/postgresMgr.js b/lib/postgresMgr.js
index 4efe456..b51b6af 100644
--- a/lib/postgresMgr.js
+++ b/lib/postgresMgr.js
@@ -1419,11 +1419,7 @@ PostgresMgr.prototype._start = function _start(cb) {
                         opsTimeout: self._opsTimeout,
                         postgresPath: self._postgresPath,
                         dataDir: self._dataDir
-                    }, 'PostgresMgr.start: start timeout');
-
-                    self._stop(function () {
-                        return callback(err, postgres);
-                    });
+                    }, 'PostgresMgr.start: start is taking a long time');
                 }
             } else {
                 log.info({
